### PR TITLE
Add install page for Cloud Foundry provider

### PR DESCRIPTION
**Pull request by fifthposition**
Also update alpha notice on CF reference page... @dorbin @lwander could you let me know if a string as I have ("1.10 and later") is acceptable or is that broken and should I think of a different way of doing this?

### CODE REVIEW

**Comment by lwander** [comment]
The alpha notice looks good to me

File: setup/install/providers/cf.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+sidebar:
+  nav: setup
+redirect_from: /setup/providers/cf/
+---
+
+{% include alpha version="1.10 and later" %}
+
+{% include toc %}
+
+In [Cloud Foundry](https://www.cloudfoundry.org) (CF), an Account maps to a user account on a CF foundation. You can add multiple accounts for one or more CF foundations.
```
Comment (dorbin) [inline setup/install/providers/cf.md]: @fifthposition 
Searching the CF docs, I can't find CF foundation other than _the_ Cloud Foundry Foundation&mdash;the holding organization of Cloud Foundry. The way you've written this sounds like a CF foundation is an part of the CF object model.

File: setup/install/providers/cf.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Prerequisites
+
+Your CF foundations' [API endpoints](https://docs.cloudfoundry.org/running/cf-api-endpoint.html) must be reachable from your installation of Spinnaker.
+
+## Adding an account
```
Comment (dorbin) [inline setup/install/providers/cf.md]: s/Adding/Add/

**Review by dorbin — COMMENTED** [review]
@fifthposition 
One small change, plus one somewhat bigger question.

**Comment by fifthposition** [comment]
Thanks for the feedback @dorbin! Updated.

**Comment by dorbin** [comment]
LGTM, @fifthposition 

@okundzich
Can you take a look, for tech review, before I approver/merge? Thanks!

**Comment by okundzich** [comment]
looks good @dorbin

**Comment by dorbin** [comment]
Thanks, @okundzich, and thanks @fifthposition

**Review by dorbin — APPROVED** [review]

**Pull request merged**